src/main/java/Iris/Parser.java

```diff
@@ -0,0 +1,72 @@
+package iris;
+
+/**
+ * The class responsible for parsing user commands in the Iris application.
+ */
+public class Parser {
+
+    /**
+     * Parses and executes user commands.
+     *
+     * @param taskStorage The storage object for tasks.
+     * @param toDoList    The to-do list containing tasks.
+     * @param input       The user input command to be parsed.
+     */
+    public void parseCommand(Storage taskStorage, ToDoList toDoList, String input) {
+        String[] sections = input.split(" ", 2);
+        String command = sections[0];
+        String rest = sections.length > 1 ? sections[1] : "";
+        switch (command) {
```

Comment:
You can consider making a command class as well as relevant subclasses. Will help the code look cleaner as the project gets bigger :)